entity-management-utils/src/main/java/eu/europeana/entitymanagement/serialization/EntityXmlSerializer.java

```diff
@@ -18,110 +12,127 @@
-	    	"         xmlns:skos=\"http://www.w3.org/2004/02/skos/core#\"\r\n" + 
-	    	"	  xmlns:ore=\"http://www.openarchives.org/ore/terms/\"\r\n" +
-	    	"         xmlns:dcterms=\"http://purl.org/dc/terms/\" >";
-    	final String XML_HEADER_TAG_ORGANIZATION = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + 
-	    	" <rdf:RDF xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\"\r\n" + 
-	    	"         xmlns:foaf=\"http://xmlns.com/foaf/0.1/\"\r\n" + 
-	    	"         xmlns:dc=\"http://purl.org/dc/elements/1.1/\"\r\n" + 
-	    	"         xmlns:edm=\"http://www.europeana.eu/schemas/edm/\"\r\n" + 
-	    	"         xmlns:vcard=\"https://www.w3.org/2006/vcard/\"\r\n" + 
-	    	"         xmlns:owl=\"http://www.w3.org/2002/07/owl#\"\r\n" + 
-	    	"	  xmlns:ore=\"http://www.openarchives.org/ore/terms/\"\r\n" +
-	    	"         xmlns:skos=\"http://www.w3.org/2004/02/skos/core#\"\r\n " +
-		"         xmlns:dcterms=\"http://purl.org/dc/terms/\" >";
-    	final String XML_END_TAG = "</rdf:RDF>";
-    	final String XML_HEADER_TAG_TIMESPAN = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + 
-		" <rdf:RDF xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" + 
-		"         xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\"\r\n" + 
-		"         xmlns:dc=\"http://purl.org/dc/elements/1.1/\"\r\n" + 
-		"         xmlns:edm=\"http://www.europeana.eu/schemas/edm/\"\r\n" + 
-		"         xmlns:foaf=\"http://xmlns.com/foaf/0.1/\"\r\n" + 
-		"         xmlns:rdaGr2=\"http://rdvocab.info/ElementsGr2/\"\r\n" + 
-		"         xmlns:owl=\"http://www.w3.org/2002/07/owl#\"\r\n" + 
-		"	  xmlns:ore=\"http://www.openarchives.org/ore/terms/\"\r\n" +
-		"         xmlns:skos=\"http://www.w3.org/2004/02/skos/core#\"\r\n" + 
-		"         xmlns:dcterms=\"http://purl.org/dc/terms/\" >";
-    
-    public String serializeXml(EntityRecord record, String profileString) throws EntityManagementRuntimeException {
-    	EntityProfile profile = EntityProfile.valueOf(profileString);
-    	String res = null;
-    	switch (profile) {
-    	case internal:
-    	    res = serializeXmlInternal(record);
-    	    break;
-    	case external:
-    	    res = serializeXmlExternal(record);
-    	    break;
-    	default:
-    	    throw new EntityManagementRuntimeException("Serialization not supported for profile:" + profile);
-    	}
-    	return res;	    	
-   	}
-
-    /**
+	/**
 	 * This method serializes EntityRecord object to xml formats for the external profile.
 	 * @param entityRecord The EntityRecord object
 	 * @return The serialized entityRecord in the xml string format
-     * @throws EntityManagementRuntimeException 
+	 * @throws EntityManagementRuntimeException
 	 */
 	public String serializeXmlExternal(EntityRecord entityRecord) throws EntityManagementRuntimeException {
-		StringWriter stringWriter = new StringWriter();
-		try {
-			XMLStreamWriter sw = xmlOutputFactory.createXMLStreamWriter(stringWriter);
+		JacksonXmlModule xmlModule = new JacksonXmlModule();
```

Comment:
I reverted the changes made to this method in #49, as the `JsonSubTypes` in `BaseEntity` caused an `XmlMapper` exception.